pyne/mesh.py

```diff
@@ -775,6 +775,14 @@ def __init__(self, mesh=None, structured=False,
                 setattr(self, name, MaterialMethodTag(mesh=self, name=name,
                         doc=doc))
 
+        # add a property: cell_cell_fracs_tag and set it to None
+        self.cell_number_tag = None
+        self.cell_fracs_tag = None
+        self.mat_name_tag = None
```

Comment:
Should take out the `mat_name_*` tags here since we aren't using them yet.

Reply:
I will delete them in next pull request.